### ISSUE TITLE
3.44.0 breaks 1GP dependency resolution in create_package_version task

### ISSUE THREAD

**Issue by georgeslater** [SFDO-Tooling/CumulusCI#2880]
### Describe the bug

After upgrading to CumulusCI 3.44.0, our `release_2gp_beta` flow started failing during the `create_package_version` task.  We have a managed package with a dependency on the NPSP (so a collection of 1GP managed packages).

Error is:
`Unable to convert dependency: Contacts & Organizations 3.19`

### Reproduction steps

Specify an NPSP dependency in cumulusci.yml

Run the following commands

```
cci org scratch release dependencies
cci task run update_dependencies --org dependencies --packages-only True
cci flow run release_2gp_beta --org dependencies
```

### Your CumulusCI and Python versions

CumulusCI version: 3.44.0 (/opt/homebrew/Cellar/cumulusci/3.44.0/libexec/bin/cci)
Python version: 3.9.7 (/opt/homebrew/opt/python@3.9/bin/python3.9)

### Operating System

Linux / MacOS

### Windows environment

- [ ] Command Prompt
- [ ] PowerShell
- [ ] Bash on Windows
- [ ] WSL

### CumulusCI installation method

brew

### Error Gist

https://gist.github.com/georgeslater/559cca8495f7ae2f42a1aabd63fc616d

**Comment by davidmreed** [comment]
Thanks for reporting, @georgeslater. I will investigate and triage today.

**Comment by davidmreed** [comment]
I've identified the issue and am working on a fix.

**Comment by davisagli** [comment]
Fixed in CumulusCI 3.44.1, which we just released.
